### ISSUE TITLE
Tasks is not working as suposed to do

### ISSUE THREAD

**Issue by Holofractographic** [FreedomCoop/valuenetwork#248]
Hi!
 When I go to Freedom Coop Membership Requests(https://ocp.freedomcoop.eu/accounting/membership-requests/)  and delete some request that was wrong then when I go to All work it still appears there: https://ocp.freedomcoop.eu/accounting/work/

**Comment by bhaugen** [comment]
Yeah, deleting the request will not automatically delete the task. It should, though - that would make the most sense. Good catch!
